packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+
+* `system.client.concurrency_rate_limit.violation`: This event is fired when the framework is in **Enforce and log per client** mode. This event is fired when a specific client, IP address, or device identifier combination makes more than two concurrent requests. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* `system.client.rate_limit.notification `: This event is fired when the framework is in **Log per client** mode. This event is fired when a specific client, IP address, or device identifier combination exceeds the total limit of 60 requests per minute. However, the end user wouldn't see a rate-limit violation. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* `system.client.concurrency_rate_limit.notification`: This event is fired when the framework is turned on in **Log per client** mode. This event is fired when a specific client, IP address, Device token combination makes more than two concurrent requests. However, the end user wouldn't see a rate-limit violation. The System Log contains information about the client ID, IP address, Device identifier, and the actual user if the user already has a valid session.
+
+### Check your rate limits with Okta Rate Limit headers
+
+The Rate Limit headers returned when client-based rate limiting is enabled are very similar to the headers returned today through the [org-wide rate limits](/docs/reference/rl-best-practices/). The difference is that the header value is specific to a given client/IP/device combination rather than the org-wide Rate Limit values. Okta provides three headers in each response to report client-specific rate limits.
+
+> **Note:** If client-based rate limiting is in **Log per client** or **No action** mode, headers that are returned still reflect the org-wide rate limits.
+
+For client-specific rate limits, three headers show the limit that is being enforced, when it resets, and how close you are to hitting the limit:
+
+* `X-Rate-Limit-Limit` - The rate limit ceiling that is applicable for the current request to the specific client/IP/device identifier combination
+* `X-Rate-Limit-Remaining` - The number of requests left for the specific client/IP/device identifier combination during the current rate-limit window
+* `X-Rate-Limit-Reset` - The time when the rate limit resets, specified in [UTC epoch time](https://www.epochconverter.com/) for the specific client/IP/device identifier combination
+
+For example:
+
+```
+HTTP/1.1 200 OK
+X-Rate-Limit-Limit: 60
+X-Rate-Limit-Remaining: 35
+X-Rate-Limit-Reset: 1516307596
+```
+
+When a specific client/IP/device identifier combination exceeds either the per minute request limit (60 requests per minute) or the concurrent limit (two concurrent requests), the `/authorize` request returns an HTTP 429 error.
```

Comment:
When a specific client/IP/device identifier combination exceeds either the **60 requests** per minute request limit or the concurrent limit (two concurrent requests), **then** the `/authorize` request returns an HTTP 429 error.